packages/fxa-settings/README.md

```diff
@@ -2,16 +2,51 @@
 
 This documentation is up to date as of 2020-08-20.
 
+## Table of Contents
+
+[Relevant ADRs](#relevant-adrs)\
+[Development](#development)\
+[GQL and REST API Calls](#gql-and-rest-api-calls)\
+— [Global Application Data](#global-application-data)\
+[Components to Know](#components-to-know)\
+— [`AlertBar`](#alertbar-and-alertexternal)\
+— [`VerifiedSessionGuard`](#sessions-and-verifiedsessionguard)\
+[Styling Components](#styling-components)\
+— [Tailwind](#tailwind)\
+—— [Component Classes](#component-classes)\
+— [PurgeCSS](#purgecss)\
+— [Custom Styles](#custom-styles)\
+[`fxa-react`](#fxa-react)\
+— [Reusing Components with `fxa-react` and Tailwind](#reusing-components-with-fxa-react-and-Tailwind)\
+[Working with SVGs](#working-with-svgs)\
+[Metrics](#metrics)\
+— [Payload Data](#payload-data)\
+— [Event Logging](#event-logging)\
+[Testing and Mocks for Tests/Storybook](#testing-and-mocks-for-testsstorybook)\
+— [`AlertBar` or `AlertExternal`](#components-that-use-alertbar-or-alertexternal)\
+— [`useAccount`](#components-that-use-useaccount)\
+[Storybook](#storybook)\
+[License](#license)
+
+## Relevant ADRs
+
+- [Create a New React Application for the Settings Redesign Project](https://github.com/mozilla/fxa/blob/main/docs/adr/0011-create-new-react-app-for-settings-redesign.md)
+- [React Toolchain for Settings Redesign](https://github.com/mozilla/fxa/blob/main/docs/adr/0013-react-toolchain-for-settings-redesign.md)
+- [Use GraphQL and Apollo for Settings Redesign](https://github.com/mozilla/fxa/blob/main/docs/adr/0016-use-graphql-and-apollo-for-settings-redesign.md)
+- [Switch from OAuth2 to Sharing sessionToken with GraphQL Server for Settings Redesign Auth](https://github.com/mozilla/fxa/blob/main/docs/adr/0017-switch-settings-auth-to-sessiontoken.md)
+- [Use Utility-First CSS (Tailwind) with Custom SCSS](https://github.com/mozilla/fxa/blob/main/docs/adr/0018-use-tailwind-with-custom-scss.md)
```

Comment:
Didn't add [auth ADR 1](https://github.com/mozilla/fxa/blob/main/docs/adr/0014-auth-for-settings-redesign.md) (superseded by auth ADR 2) in here or [CSS ADR 1](https://github.com/mozilla/fxa/blob/main/docs/adr/0015-use-css-variables-and-scss.md) (more of a foundational step for part 2) since we link to these in auth ADR 2 and CSS ADR 2. Can add them if requested.